File: 200_LeetCode_Questions/Java/D03P2_LeetCode_1281.java
class Solution {
    public int subtractProductAndSum(int n) {
        int ans, prod=1, sum=0;


        while(n>0){

            prod *= n%10;
            sum += n%10;
            n /=10;
        }	        }
        	
        ans = prod - sum;
        return ans;
            	
    }
}	

File: 200_LeetCode_Questions/C++/D03P2_LeetCode_1281.cpp
// Problem : Subtract the Product and Sum of Digits of an Integer

// Given an integer number n, return the difference between the product of its digits and the sum of its digits.

// Example 1:
// Input: n = 234
// Output: 15 
// Explanation: 
// Product of digits = 2 * 3 * 4 = 24 
// Sum of digits = 2 + 3 + 4 = 9 
// Result = 24 - 9 = 15
  
// Solution : 

class Solution
{
public:
    int subtractProductAndSum(int n)
    {
        int pod =1;
        int sod =0;
        while(n!=0)
        {
            pod *= n%10;
            sod += n%10;
            n /=10;
        }
        return pod-sod;
    }
};
